.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "conformance-tests"]
+	path = conformance-tests
```

Comment:
I expected a nested share to be put somewhere like `google-cloud-storage/test`, so each gem could be updated independently. Thoughts?

Reply:
Yes, I thought of that too, but decided to start with the simplest possible import. Really not sure what's best from the perspective of CI, please advise.

Reply:
If you have separate submodules, each gem can pin to a different commit so that each gem can be upgraded independently.

Reply:
That's a good idea. I was thinking of the more extreme approach, in which CI calls `git submodule update --remote` and we are made aware of (breaking) updates when the build fails. Otherwise, I'm afraid I probably won't remember to periodically check for changes to the tests.

Reply:
Yeah, I don't like that. Still recovering from an unbound rubocop dependency breaking our builds every time a new release was made...

Reply:
OK, I hear ya. And I agree that if pulls are going to be manual, we should definitely scope the submodules to the gems.